src/UserService.Business/Commands/User/GetUserCommand.cs

```diff
@@ -261,14 +261,11 @@ public OperationResultResponse<UserResponse> Execute(GetUserFilter filter)
       List<Guid> userImagesIds = new();
       List<ImageInfo> imagesResult = null;
 
-      if (filter.IncludeImages)
+      if (filter.IncludeImages && filter.IncludeCertificates)
```

Comment:
if !filter.IncludeCertificates user do not get other images

Reply:
if(some1)
{
    if(some2) {}
}
equal if(some1 && some 2)
i didnt write new check